app/migrations/versions/49e8a333d285_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 49e8a333d285
+Revises: d7b89a614ae9
+Create Date: 2020-06-02 10:37:49.647260
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '49e8a333d285'
+down_revision = 'd7b89a614ae9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('ix_worker_messages_table_blockchain_task_uuid', table_name='worker_messages')
```

Comment:
The OG migration seems to have been merged with the defer PR. This is just a generated migration which removes an unused index, a result of changes that happened after that one was merged